### PR TITLE
fix(1100): Use '${body}' as default for Log's EIP message

### LINKED ISSUE #1100: Use `${body}` as default for `Log`'s EIP message
### Please describe the feature that you want to propose

When adding a `Log` EIP to the canvas, there's no default value for the message. The goal for this ticket is to provide `${body}` by default as `message`.

### PR DESCRIPTION
**Pull request by shivamG640**
Fixes #1100

### CODE REVIEW

**Comment by shivamG640** [comment]
Hi @lordrip 
Is this text expected here in the message field?
image

**Comment by lordrip** [comment]
> Hi @lordrip Is this text expected here in the message field? image

Yes

**Comment by lordrip** [comment]
@shivamG640 this needs to be handled on the UI side, in the default service.

**Comment by shivamG640** [comment]
@lordrip Please review this!
Curious to know why we choose the service files to add this changes instead of the initial changes to the catalogs?

**Comment by lordrip** [comment]
> @lordrip Please review this! Curious to know why we choose the service files to add this changes instead of the initial changes to the catalogs?

Because we're not aiming to alter the catalog, we're rather providing or suggesting a useful default value.

**Review by lordrip — COMMENTED** [review]
Thanks for taking care of this @shivamG640 , it looks good, please add a unit for it and we're good to go

File: packages/ui/src/models/visualization/flows/support/camel-component-schema.service.ts [hunk 1]
```diff
@@ -364,6 +364,9 @@ export class CamelComponentSchemaService {
         break;
 
       case 'log':
+        if (typeof definition === 'object' && !isDefined(definition.message)) {
+          updatedDefinition = { ...definition, message: '${body}' };
+        }
```
Comment (lordrip) [inline packages/ui/src/models/visualization/flows/support/camel-component-schema.service.ts:369]: Hi @shivamG640,  this is the right file to add the logic: https://github.com/KaotoIO/kaoto/blob/d9986e86014ef3a6640a3f9cbc1ada333622c49f/packages/ui/src/models/visualization/flows/support/camel-component-default.service.ts#L118-124

You just need to add another case with the `message` property in it

**Comment by lordrip** [comment]
Thanks for taking care of this @shivamG640 :+1:

**Pull request merged**